File: Linguagem C/Lista_vetores_matrizes/Ex_002.c
//2) Faça um programa que declare e leia um vetor com 10 números reais. Calcule e armazene em um segundo
//vetor o quadrado de cada número real lido. Mostre os dois vetores na tela. Imprima cada vetor em uma linha,
//com os números separados entre si por um único espaço.
#include <stdio.h>
int main ()
{
    float vet[10];
    float res[10];
    int i;
    
    for(i=0; i < 10; i++)
    {
        scanf("%f", &vet[i]);
    }
    
    for(i=0; i < 10; i++)
    {
        res[i] = vet[i] * vet[i];
    }

    for(i=0; i < 10; i++)
    {
        printf("%.2f ", vet[i]);
    }
    printf("\n");

    for(i=0; i < 10; i++)
    {
        printf("%.2f ", res[i]);
    }

return 0;    
}

File: Python/006_Lista_vetores_matrizes/Ex_002.py
"""
2) Faça um programa que declare e leia um vetor com 10 números reais. Calcule e armazene em um segundo
vetor o quadrado de cada número real lido. Mostre os dois vetores na tela. Imprima cada vetor em uma linha,
com os números separados entre si por um único espaço.
"""
v1 = []

for i in range(10):
    reais = float(input(f"Digite o valor de numero {i+1}: "))
    v1.append(reais)

v2 = []  # Cria um segundo vetor para armazenar os quadrados

for num in v1:
    quadrado = num ** 2
    v2.append(quadrado)

for i in range(10):
    print(v1[i], v2[i])  # Imprime os valores dos dois vetores lado a lado
